scripts/_common.sh

```diff
@@ -12,6 +12,17 @@ pkg_dependencies="apt-transport-https postgresql postgresql-contrib php${YNH_PHP
 # PERSONAL HELPERS
 #=================================================
 
+# Assign two variables:
+#   minutes: random value between 00 and 59 (MM)
+#   hour: random value between 0 and 2
+# It is used for the cronjob, so that it runs every 3 hours starting at 00:MM, 01:MM, or 02:MM.
```

Comment:
May be we should explain more about this helper, something like :

> To avoid peak load on proxy.aneria.com, we randomize when cron job will be launched